crates/viewer/re_viewport_blueprint/src/view_contents.rs

```diff
@@ -485,7 +485,9 @@ impl<'a> DataQueryPropertyResolver<'a> {
                     override_path,
                     [&blueprint_archetypes::VisualizerOverrides::descriptor_ranges()],
                 )
-                .component_batch::<blueprint_components::VisualizerOverride>()
+                .component_batch::<blueprint_components::VisualizerOverride>(
+                    &blueprint_archetypes::VisualizerOverrides::descriptor_ranges(),
```

Comment:
sidenote, not so much related to the particular change: Have we any test that `VisualizerOverrides` (from python) still works? 🤔 
Sounds like a thing that could easily break during this whole effort

Reply:
left note in tagged component issue